docs/linux-tricks/handy-cli-apps.md

```diff
@@ -62,10 +62,6 @@ Too long; didn't read. ~~即 “太长不看”~~ 废话不多说，就是这款
 
 ### 资源
 
-- 官方网站：[https://httpie.io/cli](https://httpie.io/cli)
-- 快速入门：可使用 Homebrew 安装。输入 `tldr http` 或 `tldr https` 查看帮助，即可快速上手。
-- 相关中文教程：[https://linux.cn/article-10765-1.html](https://linux.cn/article-10765-1.html)
-
-## 附录
-
-暂时就想到这么多，到时候再添加吧。
+- 官方网站：<https://httpie.io/cli>
+- 快速入门：`http net.lolicon.app/detail` 会以 HTTP 为协议，`GET` 为方法请求这个 URL，`https` 同理。这个程序的 tldr 页面相当实用，建议执行 `tldr http` 与 `tldr httpie` 查看。
```

Comment:
I don't think it would be a good "quick start" if without installation guide. Also, `tldr httpie` is not preferred.